networking/external_dns_operator/nw-control-dns-records-public-hosted-zone-gcp.adoc

```diff
@@ -0,0 +1,153 @@
+[id="nw-control-dns-records-public-hosted-zone-gcp_{context}"]
+= Controlling DNS records on an public hosted zone for GCP by using Red Hat External DNS Operator
```

Comment:
Creating DNS records on an public hosted zone for GCP by using Red Hat External DNS Operator

Reply:
public managed zones

hosted is AWS terminology